modules/woocommerce-analytics/wp-woocommerce-analytics.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Jetpack_WooCommerce_Analytics is ported from the Jetpack_Google_Analytics code.
+ *
+ * @package Jetpack
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+require_once plugin_basename( 'classes/wp-woocommerce-analytics-utils.php' );
+require_once plugin_basename( 'classes/wp-woocommerce-analytics-universal.php' );
+
+/**
+ * Class Jetpack_WooCommerce_Analytics
+ * Instantiate WooCommerce Analytics
+ */
+class Jetpack_WooCommerce_Analytics {
+
+	/**
+	 * Instance of this class
+	 *
+	 * @var Jetpack_WooCommerce_Analytics - Static property to hold our singleton instance
+	 */
+	private static $instance = false;
+
+	/**
+	 * Instance of the Universal functions
+	 *
+	 * @var Static property to hold concrete analytics impl that does the work (universal or legacy)
+	 */
+	private static $analytics = false;
+
+	/**
+	 * WooCommerce Analytics is only available to Jetpack connected WooCommerce stores with both plugins set to active
+	 * and WooCommerce version 3.0 or higher
+	 *
+	 * @return bool
+	 */
+	public function isActiveStore() {
```

Comment:
This method may need to be declared as `static`. PHP is complaining about it being called statically but not being declared as such.

Reply:
Thanks! Fixed in https://github.com/Automattic/jetpack/pull/8296/commits/c20e2b48b3bb70a6962279023935613f8a609254